modules/blocksruntime/1.0.0/patches/add_module_and_build_file.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
+new file mode 100644
+index 000000000..83ea047de
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,41 @@
++package(default_visibility = ["//visibility:public"])
++
++cc_library(
++    name = "BlocksRuntime",
++    srcs = [
++        "BlocksRuntime/Block_private.h",
++        "BlocksRuntime/data.c",
++        "BlocksRuntime/runtime.c",
++        "config.h",
++    ],
++    hdrs = [
++        "BlocksRuntime/Block.h",
++    ],
++    copts = select({
++        "@rules_cc//cc/compiler:msvc-cl": [],
++        "//conditions:default": [
++            # BlocksRuntime/runtime.c:72:22: warning: unused function 'OSAtomicCompareAndSwapLong' [-Wunused-function]
++            "-Wunused-function",
++        ],
++    }),
++    includes = ["BlocksRuntime"],
++)
++
++# Allows the library to be referenced as @blocksruntime.
++alias(
++    name = "blocksruntime",
++    actual = "BlocksRuntime",
++)
++
++cc_test(
++    name = "sample",
++    srcs = ["sample.c"],
++    copts = ["-fblocks"],
++    # -fblocks requires clang.
++    target_compatible_with = select({
```

Comment:
*Technically* it seems that a subset of gcc-clang also works here, but I think this is fine.